client/src/components/Header/SearchBar/LocationSearch/LocationSearch.js

```diff
@@ -91,7 +90,7 @@ export default class LocationSearch extends React.Component {
         };
 
         return (
-            <div className="form-control-sm p-0 position-relative" style={{width: width, marginRight: margin, marginLeft: margin, marginBottom: barBottomSpacing}}>
```

Comment:
This stuff is now all directly applied in `Header.scss` so we don't need to pollute the javascript with style information